backend/gateway/mux/mux.go

```diff
@@ -182,8 +182,11 @@ func customResponseForwarder(ctx context.Context, w http.ResponseWriter, resp pr
 			}
 			code = headerCodeOverride
 		}
+
+		w.Header().Set("Location", redirects[0])
```

Comment:
Can this ever be more than 1?

Reply:
The effective value for headers is the first one, e.g. https://github.com/golang/go/blob/cd28ca3a4e130c88dd8f2d198050620b11d22ce4/src/net/textproto/header.go#L38

Additional values are ignored. But in our case no, there should never be any additional values anyways.